docs/src/intervention.html.jinja2

```diff
@@ -685,6 +747,82 @@
+                                    <option value="ilp">Invasive adenocarcinoma, Lepidic predominant adenocarcinoma</option>
+                                    <option value="iap">Invasive adenocarcinoma, Acinar predominant adenocarcinoma</option>
+                                    <option value="ipp">Invasive adenocarcinoma, Papillary predominant adenocarcinoma</option>
+                                    <option value="imp">Invasive adenocarcinoma, Micropapillary predominant adenocarcinoma</option>
+                                    <option value="isp">Invasive adenocarcinoma, Solid predominant adenocarcinoma</option>
+                                    <option value="imdp">Invasive adenocarcinoma, multiple predominant patterns</option>
+                                    <option value="ipcd">Invasive adenocarcinoma, Predominant subtype cannot be determined</option>
+                                    <option value="im">Invasive mucinous adenocarcinoma</option>
+                                    <option value="m">Mixed invasive mucinous and nonmucinous adenocarcinoma</option>
+                                    <option value="fa">Fetal adenocarcinoma</option>
+                                    <option value="ea">Enteric adenocarcinoma</option>
+                                    <option value="ca">Colloid adenocarcinoma</option>
+                                    <option value="srf">Signet ring features</option>
+                                    <option value="ccf">Clear cell features</option>
+                                    <option value="nf">NSLC, favor adenocarcinoma</option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="col-md-7">
+                            <div class="table-responsive">
+                                <table class="table table-striped table-bordered">
+                                    <tr>
+                                        <td><label>Predominant features</label></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamds", "s","solid") }}</div></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamda", "a","acinar") }}</div></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamdp", "p","papillary") }}</div></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamdmp", "m","micropapillary") }}</div></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamdb", "b","Lepidic/ bronchioloalveolar") }}</div></td>
+                                        <td><div class="form-group">{{ f.checkbox("pamdo", "o","other (specify)") }}<input name="pamdos" id="pamdos" class="form-control" size="8"></div></td>
```

Comment:
These other-specify fields need to be conditionally displayed. Like I said in a comment earlier. These would make no sense to allow a user to say "solid" and also be able to type into the "other" text box.